src/posix/threading.c

```diff
@@ -176,7 +204,7 @@ sys_thread_t *sys_thread_self(void)
 }
 
 // note: called by signal handler, do not invoke async signal unsafe code in this function!  See man signal(7)
-const char *sys_thread_name(sys_thread_t *thread)
+const char *sys_thread_name(const sys_thread_t *thread)
```

Comment:
A pedestrian programmer like me would think that there would be a return statement in this function but I don't see any.

Reply:
It's there: see line 211